wasm/tx_template/Cargo.lock

```diff
@@ -343,9 +343,9 @@ checksum = "bef38d45163c2f1dde094a7dfd33ccf595c92905c8f8f4fdc18d06fb1037718a"
 
 [[package]]
 name = "bitvec"
-version = "1.0.0"
+version = "1.0.1"
```

Comment:
When I resynced to match `namada` branch's dependencies, it also updated any dependencies which are only in `eth-bridge-integration` (eg this `bitvec` dependency, haven't seen any others)